server/src/Model/User/UserSchema.js

```diff
@@ -0,0 +1,11 @@
+import { Schema, model } from 'mongoose';
+
+const userSchema = new Schema({
+  name: String,
+  email: String,
+  password: String,
+});
+
+const userModel = model('UserSchema', userSchema);
```

Comment:
Put all the string constants in separate file, so that you have to update at a single place

Reply:
Done